rest/server_context.go

```diff
@@ -924,11 +924,12 @@ func (sc *ServerContext) asyncDatabaseOnline(nonCancelCtx base.NonCancellableCon
 		return
 	}
 
-	base.DebugfCtx(ctx, base.KeyConfig, "Async database initialization complete, starting online processes...")
+	base.InfofCtx(ctx, base.KeyAll, "Async database initialization complete, starting online processes...")
 	err := dbc.StartOnlineProcesses(ctx)
 	if err != nil {
-		base.InfofCtx(ctx, base.KeyAll, "Error starting online processes after async initialization: %v", err)
+		base.WarnfCtx(ctx, "Error starting online processes after async initialization: %v", err)
```

Comment:
Arguably this should be an error instead of just a warning - the database failed to come online and it will require user-interaction to either retry or correct?

Reply:
Bumped to Error